Common/src/main/java/com/tabnineCommon/binary/BinaryRequestFacade.java

```diff
@@ -37,9 +37,13 @@ public <R extends BinaryResponse> R executeRequest(BinaryRequest<R> req, int tim
               .get(timeoutMillis, TimeUnit.MILLISECONDS);
 
       if (result != null) {
+        // I think this is a bug. we have requests that are fine to return null (EmptyResponse is
+        // null...)
+        // such as login/logout requests
```

Comment:
Don't comment it here, open a ticket and talk with me & @yanir-codota maybe - let's check this out